src/renderer/store/modules/profiles.js

```diff
@@ -29,7 +29,14 @@ const getters = {
   profileById: (state) => (id) => {
     const profile = state.profileList.find(p => p._id === id)
     return profile
-  }
+  },
+
+  getSubscribedChannelIdSet: (state) => {
+    const mainProfile = state.profileList.find((profile) => {
+      return profile._id === MAIN_PROFILE_ID
+    })
```

Comment:
The all channels profile is always the first profile in the array, so you don't have to search through the array for it :)
```suggestion
    const mainProfile = state.profileList[0]
```